src/DssSpell.sol

```diff
@@ -44,109 +40,150 @@ contract DssSpellAction is DssAction {
+        (,,,uint256 lineReduction,) = vat.ilks("PSM-PAX-A");
+
+        // Remove PSM-PAX-A from Autoline.
+        DssExecLib.removeIlkFromAutoLine("PSM-PAX-A");
 
-        // Launch Project - 4,500,000.00 DAI - 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F
-        // Launch Project - 820.00 MKR - 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F
+        // Set PSM-PAX-A debt ceiling to 0
+        DssExecLib.setIlkDebtCeiling("PSM-PAX-A", 0);
 
-        // ---------- Update doc parameter ----------
-        // Forum: https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/14
+        // Reduce Global Debt Ceiling? Yes
+        vat.file("Line", vat.Line() - lineReduction);
 
-        // Update HVBank (RWA009-A) doc to QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT
-        _updateDoc("RWA009-A", "QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT");
+        // ---------- RETH-A Offboarding Parameters Finalization ----------
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-8/23445
 
-        // ---------- Spark D3M line increase ----------
-        // Forum: https://forum.makerdao.com/t/spark-spell-proposed-changes/23298
-        // Poll: https://vote.makerdao.com/polling/QmdQSuAc
+        // Set chop to 0%.
+        DssExecLib.setIlkLiquidationPenalty("RETH-A", 0);
 
-        // Increase the line by 400 million from 800 million to 1.2 billion Dai
-        DssExecLib.setIlkAutoLineDebtCeiling("DIRECT-SPARK-DAI", 1200 * MILLION);
+        // Set tip to 0%
+        DssExecLib.setKeeperIncentiveFlatRate("RETH-A", 0);
+
+        // Set chip to 0%
+        DssExecLib.setKeeperIncentivePercent("RETH-A", 0);
+
+        // Set Liquidation Ratio to 10,000%.
+        // Note: We are using low level methods because DssExecLib only allows setting `mat < 1000%`: https://github.com/makerdao/dss-exec-lib/blob/69b658f35d8618272cd139dfc18c5713caf6b96b/src/DssExecLib.sol#L717
+        DssExecLib.setValue(MCD_SPOT, "RETH-A", "mat", 100 * RAY);
+
+        // Note: Update collateral price to propagate the changes
+        DssExecLib.updateCollateralPrice("RETH-A");
+
+        // ---------- SBE parameter changes ----------
+        // Forum: https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-4/23441
+
+        // Increase bump by 20,000, from 30,000 to 50,000 DAI
+        DssExecLib.setValue(MCD_VOW, "bump", 50 * THOUSAND * RAD);
+
+        // Increase hop by 10,512, from 15,768 to 26,280 Seconds
+        DssExecLib.setValue(MCD_FLAP, "hop", 26_280);
+
+        // ---------- Approve HV Bank (RWA009-A) DAO Resolution ----------
+        // Forum: https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/21
+
+        // Mainnet - Add RWA009_A_INPUT_CONDUIT_URN_USDC deployed at 0x08012Ec53A7fAbf6F33318dfb93C1289886eBBE1 to the chainlog
+        // Note: skipped on goerli as instructed
+
```

Comment:
nit, do we want to add `Call <conduit>.rely(MCD_ESM) to allow ESM module to deny the pause proxy in SwapInputConduit contracts` with `Note` comment as well?

Reply:
Those duplicative instructions are quite confusing, but you're right, I probably need to explain why the second one isn't taken.